Persistence/Migrations/20230728184205_TransactionsBelongToFinancialYear.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Persistence.Migrations;
+
+/// <inheritdoc />
+public partial class TransactionsBelongToFinancialYear : Migration
+{
+    /// <inheritdoc />
+    protected override void Up(MigrationBuilder migrationBuilder)
+    {
+        migrationBuilder.Sql("INSERT INTO " +
+                             "HaspMan.FinancialYears (Id, StartDate, EndDate, IsClosed) " +
+                             "VALUES (newId(), CAST('2022-09-01 00:00:00.0000000 +02:00' AS DATETIMEOFFSET),CAST('2023-08-31 00:00:00.0000000 +02:00' AS DATETIMEOFFSET), 0)");
+
+        migrationBuilder.DropIndex("IX_Transactions_FinancialYearId", schema: "HaspMan", table: "Transactions");
+        migrationBuilder.AlterColumn<Guid>(
+            "FinancialYearId", 
+            schema:"HaspMan",
+            table: "Transactions",
+            nullable: false);
+        migrationBuilder.CreateIndex(
+            name: "IX_Transactions_FinancialYearId",
+            schema: "HaSpMan",
+            table: "Transactions",
+            column: "FinancialYearId");
```

Comment:
This only sets the field to non-nullable in the index, however the table should be altered as well